routes/web.php

```diff
@@ -72,6 +72,10 @@
 
     Route::resource('dm-entry', \App\Http\Controllers\DMEntryController::class)
         ->only('index');
+
+    Route::resource('attach-entry-to-character', App\Http\Controllers\CharacterBulkAttachDMEntryController::class)->parameters([
```

Comment:
Please add documentation for this route to the wiki whenever you can :)

Reply:
Hey @IanjhPhillips, can you confirm that the documentation follows your template?

Reply:
just checked and it fits perfectly. nice work!